src/test/ui/pattern/const-pat-ice.stderr

```diff
@@ -1,4 +1,4 @@
-thread 'rustc' panicked at 'assertion failed: rows.iter().all(|r| r.len() == v.len())', src/librustc_mir/hair/pattern/_match.rs:1084:5
```

Comment:
hmm... I could make compiletest detect ICEs automatically and filter the line/col info